client/src/DashBoardSidebar.js

```diff
@@ -0,0 +1,85 @@
+import DashboardHome from './DashboardHome';
+import DashboardCustomers from './DashboardCustomers';
+import DashboardDispatchers from './DashboardDispatchers';
+import { Switch, Route, useRouteMatch, Link } from 'react-router-dom';
+import profile from './assets/Profile.png';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+function Dashboard() {
+  let match = useRouteMatch();
+
+  return (
+    <>
+      <section className="flex h-9/10">
+        {/* @TODO nav should be made a component */}
+        <nav className="border-r-2 border-gray-300 w-1/4 ">
+          <div className="flex items-center px-14 py-5">
+            <img src={profile} className="rounded-full h-10 w-10 mr-4" />
+            <Link to={`${match.url}`}>Miroslav Minev</Link>
+          </div>
+
+          <ul className="flex flex-wrap flex-col content-center">
+            <li className="mb-9">
+              <Link to={`${match.url}`}>Overview</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/analyzer`}>Traffic analyzer</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/registered-user`}>Registered users</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/packages`}>Created packages</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/payments`}>Payments</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/feedback`}>Feedback</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/staffs`}>Staffs</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/calendar`}>Calendar</Link>
+            </li>
+            <li className="mb-9">
+              <Link to={`${match.url}/`}>LogOut</Link>
+            </li>
+          </ul>
+        </nav>
```

Comment:
This is the only part I expect to be in this file, all other codes you moved here aren't sidebar. Also, rename this file to `DashboardSidebar.js` and not `DashBoardSidebar.js`. `Dashboard` is a single word you don't need to put camel case in-between.